lib/net/ldap.rb

```diff
@@ -27,6 +27,14 @@ class LDAP
 require 'net/ldap/connection'
 require 'net/ldap/version'
 require 'net/ldap/error'
+require 'net/ldap/auth_adapter'
+require 'net/ldap/auth_adapter/simple'
+require 'net/ldap/auth_adapter/sasl'
+require 'net/ldap/auth_adapter/gss_spnego'
```

Comment:
Can we leave this adapter out of this PR since it's not one of the existing adapters we currently support? After this merges, we can create a separate gem and link to it from the README.